tools/postgres/purge_ttl.py

```diff
@@ -0,0 +1,196 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import argparse
+import logging
+import os
+import sys
+from datetime import datetime
+from typing import List, Optional, Tuple
+from urllib.parse import urlparse
+
+import sqlalchemy
+from sqlalchemy import text
+from statsd.defaults.env import statsd
+
+logging.basicConfig(
+    format='{"datetime": "%(asctime)s", "message": "%(message)s"}',
+    stream=sys.stdout,
+    level=logging.INFO,
+)
+
+
+def get_db_engine(database_url: str):
+    """Create a SQLAlchemy engine from a database url."""
+    parsed = urlparse(database_url)
+    if parsed.scheme not in ['postgresql', 'postgres']:
+        raise ValueError(f"Unsupported database scheme: {parsed.scheme}")
+
+    # newer versions of SQLAlchemy want 'postgresql' instead of 'postgres'
+    if parsed.scheme == 'postgres':
+        database_url = database_url.replace('postgres://', 'postgresql://', 1)
+
+    return sqlalchemy.create_engine(database_url)
```

Comment:
parsed url has a nice method for this
```suggestion
    # newer versions of SQLAlchemy want 'postgresql' instead of 'postgres'
    if parsed.scheme == 'postgres':
        parsed = parsed._replace(scheme='postgresql')

    return sqlalchemy.create_engine(parsed.geturl())
```